lib-dmtf/model/networkadapter.go

```diff
@@ -35,6 +37,7 @@ type NetworkAdapter struct {
 	Model                  interface{}            `json:"Model"`
 	NetworkDeviceFunctions *Link                  `json:"NetworkDeviceFunctions,omitempty"`
 	NetworkPorts           *Link                  `json:"NetworkPorts,omitempty"`
+	Oem                    *Oem                   `json:"Oem,omitempty"`
```

Comment:
update Oem to interface type instead of Oem sturct

Reply:
Oem is not struct, it's of type interface{}